src/tutorials/0005-regular-files-api/06.md

```diff
@@ -8,13 +8,13 @@ Now that we've wrapped some files in a directory in our IPFS node, let's learn h
 You can call the `ls` method like so:
```

Comment:
There's nothing here to introduce the fact that Async Iterables are involved but I got an error message stating I was wrong because of them. :)